src/config/work_mode.rs

```diff
@@ -2,8 +2,9 @@ use std::str::FromStr;
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum WorkMode {
-    Normal,     //generate widgets etc.
-    Sys,        //generate -sys with ffi
+    Normal,     // generate widgets etc.
+    Sys,        // generate -sys with ffi
+    Doc,        // generate documentation file
```

Comment:
Can you add `doc` to https://github.com/gtk-rs/gir/blob/master/src/config/config.rs#L22 ?